### ISSUE TITLE
error when opening

### ISSUE THREAD

**Issue by thaythayt0722** [palahsu/DDoS-Ripper#84]
>>> DRipper.py
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'DRipper' is not defined
>>>

**Comment by Ahmad-Imran5374** [comment]
yeah, same issue with me, but now working with python 3.8 version
